### PR TITLE
ES-DE.EmulationStation-DE version 2.0.1

### PR DESCRIPTION
**Pull request by superusercode**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107333)

### CODE REVIEW

File: manifests/e/ES-DE/EmulationStation-DE/2.0.1/ES-DE.EmulationStation-DE.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ES-DE.EmulationStation-DE
+PackageVersion: 2.0.1
+PackageLocale: en-US
+Publisher: EmulationStation-DE
+PackageName: EmulationStation-DE
```
Comment (stephengillie) [inline manifests/e/ES-DE/EmulationStation-DE/2.0.1/ES-DE.EmulationStation-DE.locale.en-US.yaml:8]:
```suggestion
PackageName: EmulationStation-DE
PackageUrl: https://www.es-de.org/
```

**Comment by stephengillie** [comment]
Manual Validation failed with:
> Faulting application name: EmulationStation.exe, version: 2.0.1.0, time stamp: 0x6444f41e 
> Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000 Exception code: 0xc0000005 
> Fault offset: 0x0000000000000000 Faulting process id: 0x0x1d44 
> Faulting application start time: 0x0x1d98a868f642181 
> Faulting application path: C:\Program Files\EmulationStation-DE\EmulationStation.exe 
> Faulting module path: unknown Report Id: 4216374c-3579-46b0-8ec4-5c090547a3fd 
> Faulting package full name: 
> Faulting package-relative application ID:

**Comment by stephengillie** [comment]
> @stephengillie is there a tracking issue for upgrading the testing VM hardware?

Nor ro my knowledge.

**Comment by Trenly** [comment]
This is what I get on a clean sandbox -
image

**Comment by Trenly** [comment]
> Does the sandbox not have game bar installed? That's not a deletable inbox app on normal windows installs
> image

Windows sandbox (and presumably the pipelines) don't have the game bar

**Comment by stephengillie** [comment]
> Does the sandbox not have game bar installed? That's not a deletable inbox app on normal windows installs image

image
Yes, it's present on Manual pipeline VMs.

**Comment by stephengillie** [comment]
> Manual Validation failed with:
> 
> > Faulting application name: EmulationStation.exe, version: 2.0.1.0, time stamp: 0x6444f41e
> > Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000 Exception code: 0xc0000005
> > Fault offset: 0x0000000000000000 Faulting process id: 0x0x1d44
> > Faulting application start time: 0x0x1d98a868f642181
> > Faulting application path: C:\Program Files\EmulationStation-DE\EmulationStation.exe
> > Faulting module path: unknown Report Id: 4216374c-3579-46b0-8ec4-5c090547a3fd
> > Faulting package full name:
> > Faulting package-relative application ID:

Had the same in a Windows 10 VM.

**Comment by stephengillie** [comment]
> still happen under a win11 vm?

Yes, in the first attempt. This second attempt was a re-run after re-adding Windows 10 to the Manual Validation pipeline.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**